src/components/header/header.js

```diff
@@ -0,0 +1,298 @@
+
+export default class HeaderComponent extends HTMLElement {
+    constructor() {
+        super();
+
+        const bodyElement = document.body;
+        bodyElement.style.margin = '0';
+        this.attachShadow({ mode: 'open' });
+        this.shadowRoot.innerHTML = `
+        
+        <header>
+            <div class="logo-bar">
+                <img class="greenwood-logo" src="/assets/greenwood-logo.svg" alt="Logo">
```

Comment:
I notice this isn't the same logo from our old site, which I think was what you mentioned, so let me make a note of that and try and get the original source, though worst case scenario we can just re-use the images for now.